src/components/ErrorMessages/ErrorMessages.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import { TodosContext } from '../TododsContext/TodosContext';
+
+export const ErrorMessages: React.FC = () => {
+  const { errorMessage, setErrorMessage } = useContext(TodosContext);
+
+  let isErrorHidden = true;
```

Comment:
It's not recommended to use stateful variables outside of the state. Consider using the useState hook for isErrorHidden variable.